.github/workflows/ci.yml

```diff
@@ -3,60 +3,31 @@ on: [push, pull_request]
 jobs:
   test-elixir:
     runs-on: ubuntu-latest
-    container:
-      image: hexpm/elixir:${{ matrix.elixir }}-erlang-${{ matrix.otp }}-alpine-3.12.0
-      env:
-        ELIXIR_ASSERT_TIMEOUT: 2000
-
+    env:
+      GIT_URL_INSTEAD_OF: "https://github.com/=git@github.com:"
+      FORCE_COLOR: 1
+    
     strategy:
       fail-fast: false
       max-parallel: 1
       matrix:
         include:
-          - elixir: 1.9.4
```

Comment:
It seems that there are a couple of tests that are flakey on these versions, when run in earthly, at least in my testing.  Without them, the tests pass consistently.